### DISCUSSION TITLE
Compressed File Support

### DISCUSSION THREAD

**Discussion by trymeouteh** [PaulWoitaschek/Voice#1348] [Ideas]
Please add support for compressed files such as 7Z files. This way users can take all the MP3 files of an audiobook and place them inside of a compressed 7Z file and have only one file to manage an entire audiobook and have it take up less storage space.

- Compressed file make the audiobook file smaller in size
- Easy to edit (Extract archive, edit file with tag editor, recompress)
- Can take MP3 files and wrapped them into one file (Easier to do than converting MP3 files into an M4B)

This will be easier to manage. M4B look great but there very few programs out there that allow us to edit M4B file for its chapters and tags, making it hard to have an organised audiobook library.

**Comment by darksidemilk** [comment]
There's a program called open audible that can join mp3 files and maintains the joined files in tracks when played in voice. The joining of audio files can be done with a free version. Just offering a possible workaround.

**Reply by darksidemilk** [reply]
I also found a program that isn't developed anymore called markable that utilizes iTunes for ripping and then creates m4b joined files.
